recipe/meta.yaml

```diff
@@ -14,20 +14,18 @@ package:
 
 source:
   # latest commit from branch {{ cctools_version }}-ld64-{{ ld64_version }}
-  - url: https://github.com/tpoechtrager/cctools-port/archive/04663295d0425abfac90a42440a7ec02d7155fea.tar.gz
-    sha256: 70a7189418c2086d20c299c5d59250cf5940782c778892ccc899c66516ed240e
+  - url: https://github.com/tpoechtrager/cctools-port/archive/437ced391dbf14dce86f977ca050a750d5682f39.tar.gz
+    sha256: cf4020ddb92467de020c322eb13f0cac9ed992daa862a09bc2607e42467b3042
     patches:
       - patches/0001-Don-t-link-with-libc-abi.patch
       - patches/0002-ld64-add-conda-specific-env-vars-to-modify-lib-searc.patch
       # Taken from https://github.com/tpoechtrager/cctools-port/pull/116
       - patches/0003-Restore-using-__builtin_available-for-macos.patch
       # Taken from https://github.com/tpoechtrager/cctools-port/pull/118
       - patches/0004-Turn-off-outputIsMappableFile-when-building-to-osx-a.patch
```

Comment:
Sidenote: There's some (belated) upstream review comments for both of these PRs.